v4: Add an actual `data-touch="false"` example in the carousel docs (#32639)
twbs/bootstrap: site/content/docs/4.5/components/carousel.md

</div>
{{< /example >}}

> ### Suppressing touch swiping

Reviewer: ```suggestion
### Disable touch swiping
```

> By default, carousels support swiping left/right on touchscreen devices to move between slides. This can be explicitly suppressed using the `data-touch` attribute. The example below also does not include the `data-ride`, and has `data-interval="false"`, meaning that it won't autoplay.

Reviewer: ```suggestion
Carousels support swiping left/right on touchscreen devices to move between slides. This can be disabled using the `data-touch` attribute. The example below also does not include the `data-ride` and has `data-interval="false"` so it doesn't autoplay.
```

Author: addressed with separate commit, as i also noticed that this could do with the extra word "attribute" after `data-ride`
